### PR TITLE
Rename occurrences of CVC4 to CVC5.

### PR DESCRIPTION
**Pull request by aniemetz**
This renames everything but GitHub links and build system related
macros. Switching the build system to cvc5 will be the last step in the
renaming process.

### CODE REVIEW

File: src/util/sexpr.cpp [hunk 1]
```diff
@@ -279,7 +279,7 @@ bool SExpr::languageQuotesKeywords(OutputLanguage language) {
       return true;
     case language::output::LANG_AST:
     case language::output::LANG_CVC3:
-    case language::output::LANG_CVC4:
+    case language::output::LANG_CVC:
```
Comment (mpreiner) [inline src/util/sexpr.cpp:282]: Not sure if this should really be renamed. @ajreynol  what do you think?

Comment (aniemetz) [reply src/util/sexpr.cpp:282]: Do we even still support LANG_CVC3 @ajreynol ?

Comment (ajreynol) [reply src/util/sexpr.cpp:282]: I think LANG_CVC3 can be deleted, it doesn't appear to be used.

Comment (aniemetz) [reply src/util/sexpr.cpp:282]: Resolved offline, yes we can rename and yes we can remove LANG_CVC3. Will do the latter in a separate PR.

**Review by mpreiner — APPROVED** [review]

**Pull request merged**